kubectl/taskrun.yaml

```diff
@@ -6,16 +6,30 @@ spec:
   taskRef:
     name: kubectl-deploy-pod
   params:
+  - name: action
+    value: create
```

Comment:
When action is `GET` where can we mention the object name 

Ex: 
For creating pod we can mention action as `create` and yaml content in `manifest` section
similarly for `GET`, `DELETE` where do we mention which resource needs to be deleted

Reply:
For now, all `action` need a `manifest`, so you need supply `manifest` for `Get` and `Delete`, maybe with a simplified version, maybe with `ObjectReference` only.

Reply:
Can we have a one more task run specifying GET or DELETE operation with manifest
It would be easy to understand how to use task when we wanted to do GET or DELETE operation

Reply:
Agree with you, thanks